src/components/product-tile/ProductTile.tsx

```diff
@@ -48,9 +51,19 @@ const ProductTileContainer = styled.li`
 
 export const ProductTile: React.FC<{ product: Product }> = ({ product }) => {
   const { t } = useTranslation();
+  const { cart, setCart } = useContext(CartContext);
 
   const handleClick = () => {
-    // TODO: Add to cart
+    const productInCart = cart.find((item) => item.id === product.id);
+
+    if (productInCart) {
+      changeQuantity(product.id, 1, setCart);
+    } else {
+      setCart((currCart) => [
```

Comment:
You could create an addToCart function as well in utils for this, it will be useful when we can add it from the product details page